### PR TITLE
docs: add manim latex chocolatey package

### PR DESCRIPTION
**Pull request by naveen521kk**
## List of Changes
- Add a small follow up documentation.

## Motivation
https://chocolatey.org/packages/manim-latex is made by me for manim.

## Acknowledgement
- [x] I have read the [Contributing Guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP))

### CODE REVIEW

File: docs/source/installation/win.rst [hunk 1]
```diff
@@ -92,26 +108,6 @@ Using MiKTex
 2. You can check if you did it right by running ``refreshenv`` to update your
    environment variable and running ``latex``.
 
-Using TinyTex
```
Comment (leotrs) [inline docs/source/installation/win.rst:95]: So we're no longer using TinyTex at all?

Comment (naveen521kk) [reply docs/source/installation/win.rst:95]: Pretty much both are same.

Comment (naveen521kk) [reply docs/source/installation/win.rst:95]: @leotrs Is this PR waiting for this?

**Review by leotrs — APPROVED** [review]

**Review by behackl — APPROVED** [review]
Changes and the built documentation LGTM. 👍

**Pull request merged**